autogtp/Game.cpp

```diff
@@ -30,7 +30,7 @@ Game::Game(const QString& weights, const QString& opt) :
     m_moveNum(0)
 {
 #ifdef WIN32
-    m_cmdLine.append(".exe");
+    m_cmdLine = "leela-zero.exe";
```

Comment:
Would rather not change this, i.e. keep it the same between Unix/Win and only add the right extension.

Reply:
Will change the binary name to make it matching.